### ISSUE TITLE
arrows for carousel

### ISSUE THREAD

**Issue by ngisolfi** [autonlab/autonlab.github.io#25]
is it possible to add forward back arrows to iterate through the carousel manually?

**Comment by betababb** [comment]
yes! i think it is typically done using javascript, but [this](https://getbootstrap.com/docs/4.0/components/carousel/) css bootstrap should be easy to implement as well
